addons/hr_attendance/models/hr_employee.py

```diff
@@ -204,6 +204,5 @@ def read_group(self, domain, fields, groupby, offset=0, limit=None, orderby=Fals
     def _compute_presence_icon(self):
         res = super()._compute_presence_icon()
         # All employee must chek in or check out. Everybody must have an icon
-        employee_to_define = self.filtered(lambda e: e.hr_icon_display == 'presence_undetermined')
-        employee_to_define.hr_icon_display = 'presence_to_define'
+        self.filtered(lambda employee: not employee.show_hr_icon_display).show_hr_icon_display = True
```

Comment:
Is this correct?

Because if a record matches `not show_hr_icon_display` then it means it's already True :smile: